clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3853,6 +3853,36 @@ def MemCpyOp : CIR_Op<"libc.memcpy"> {
     mlir::cir::IntType getLenTy() { return getLen().getType(); }
   }];
 }
+//===----------------------------------------------------------------------===//
+// MemSetOp
+//===----------------------------------------------------------------------===//
+
+def MemSetOp : CIR_Op<"libc.memset"> {
+  let arguments = (ins Arg<VoidPtr, "", [MemWrite]>:$dst,
+                       UInt8:$val,
```

Comment:
`libc` operations are 1:1 correspondance to libc functions. The second parameter of `memset` has type `int` and we should mimic that:

```suggestion
                       SInt32:$val,
```